src/creature.h

```diff
@@ -544,6 +544,9 @@ class Creature
         virtual const std::string &symbol() const = 0;
         virtual bool is_symbol_highlighted() const;
 
+		// Turn since creature was milked for last time
+		int turn_next_milking = 0;
```

Comment:
I don't know if it would be better to make some other animal recharge its milk faster or just make it a once per day thing. Or even if make it configurable on the json.